people-and-planet-ai/geospatial-classification/requirements.txt

```diff
@@ -1,8 +1,6 @@
 numpy==1.22.0; python_version > "3.6"
 tensorflow==2.8.1
 earthengine-api==0.1.311
-pytest==7.0.1
```

Comment:
Removed these since they're already included in the requirements-test.txt so they were redundant here.